### ISSUE TITLE
Film players?

### ISSUE THREAD

**Issue by softsound** [josefnpat/roguecraft-squadron#190]
It's a good way to stay motivated.

**Comment by josefnpat** [comment]
Why is this 

1) a question
2) a feature request
3) marked as research?

We will need to get a release form if we want to use any of it for ads

updating ticket accordingly

**Comment by snowschu** [comment]
We have done two official local recordings but haven't looked into the 'results' of the recordings.

------
Ideally, I'd like to continue to have recordings to work off of, especially for bugs and crashes that happen during demoing.

I would also like to encourage stronger note taking during demoing.

**Comment by josefnpat** [comment]
Perhaps don't lose the notes :3

**Comment by snowschu** [comment]
Found footage (it's on Demo machine Ash #1).

----------
If we continue to record players, I will need to add the same settings to Demo Machine Ash #2. (We're doing multiplayer. There's no reason to not record both players). 

We could use the footage as b-roll for future demo videos or trailers. This would require us to lose the mouse and keyboard overlay data/image (and any other data we have or would want on top of the recording).  

My personal preference is that we continue to use the recordings as playtesting feedback.

**Comment by snowschu** [comment]
> We could use the footage as b-roll for future demo videos or trailers. This would require us to lose the mouse and keyboard overlay data/image (and any other data we have or would want on top of the recording).

We technically don't _need_ to lose the keyboard/mouse capture. There's no reason for the footage to use the full view of the game. I could also make it take up less space on the screen. 

In some shots of the game, however, we'll need to have a wide shot of the game. This can be done with additional recordings (from us).

**Comment by josefnpat** [comment]
If we do another recording session, it needs to have a purpose (e.g. marketing or QA or UX testing)

**Comment by snowschu** [comment]
We have filmed players during the demos we attended. Ultimately, though, we didn't put it to good use and the videos tended to be out of date.

In the future, it would be great to continue to get footage, specially when there's game breaking issues or if a player does something super awesome and we can use it for promotion.

Things blocking this:
- The demo machines are used as work machines and are not available until the day of the demos. This is just a fact and not a judgement.
- No real workflow to getting these set up and working correctly the first time.

I will be closing ticket. We can reopen it if we want to revisit this again in the future.
